pkg/crds/crds.go

```diff
@@ -170,6 +170,10 @@ func GetCRDs(variant opv1.ProductVariant) []*apiextenv1.CustomResourceDefinition
 	// our original copy of the definitions are not tainted with a ResourceVersion
 	copy := []*apiextenv1.CustomResourceDefinition{}
 	for _, crd := range crds {
+		// Skip the Tenant CRD - this is only used in Calico Cloud.
+		if crd.Name == "tenants.operator.tigera.io" {
+			continue
+		}
```

Comment:
Will it have any impact for cloud since we are skipping "tenant crds" in the generic getCRD function? @caseydavenport ?

Reply:
I don't think so